winfsp/src/filesystem/context.rs

```diff
@@ -67,18 +67,18 @@ pub trait FileSystemContext: Sized {
     ///    Ok(security)
     /// }
     /// ```
-    fn get_security_by_name<P: AsRef<U16CStr>>(
+    fn get_security_by_name(
```

Comment:
Please drop your changes to `FileSystemContext` in [190fa78](https://github.com/SnowflakePowered/winfsp-rs/pull/9/commits/190fa784b5f29aaa43c06fd9c89ddc6f9682904a). It is intentionally `AsRef<U16CStr>>` to allow for potentially lossless conversions between string type representations such like `HSTRING`

Reply:
When do you need to call these methods, except in the interface ? 🤔